tests/security/apparmor/aa_enforce.pm

```diff
@@ -0,0 +1,48 @@
+# Copyright (C) 2018 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+#
+# Summary: Enforce a disabled profile with aa-enforce
+# Maintainer: Wes <whdu@suse.com>
+# Tags: poo#36877, tc#1621145
+
+use strict;
+use base "consoletest";
+use testapi;
+use utils;
+
+sub run {
+    select_console 'root-console';
+
+    systemctl('restart apparmor');
+
+    validate_script_output "aa-disable usr.sbin.ntpd", sub {
+        m/Disabling.*ntpd/;
+    };
+
+    # Check if /usr/sbin/ntpd is really disabled
+    die "/usr/sbin/ntpd should be disabled"
+      if (script_run("aa-status |grep /usr/sbin/ntpd") == 0);
```

Comment:
Missed that it won't work on SLE/Leap 15 and TW as ntpd got replaced with chronyd. I guess we can extract logic we have in https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/console/yast2_ntpclient.pm#L28

Reply:
ntpd is still available on SLE/Leap 15 and TW, but the package is not installed by default. chronyd is the default one, but there is no available apparmor profile yet. So I add a zypper_call to install ntp package without considering whether it has already installed or not.

Reply:
That's good point, so for openSUSE we are safe, but I've checked for SLE15 and in case of default SLES installation which is basesystem and server applications modules, it's not available. It's only in legacy. So for SLE15 we need to enable it before the installation (see https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/registration.pm#L126). And again sorry for raising this point this late.

Reply:
Okay, then now I changed to use nscd which is in the base system and installed by default for both SLE/openSUSE. Since it is for the tool testing, not the profile, I think nscd is suitable.